# Flag used to start the navigation when target IDs are received and to stop it when all the target IDs have been found
bool status 